### ISSUE TITLE
Can this method be applied for text classification task?

### ISSUE THREAD

**Issue by sanazjb** [asappresearch/aum#5]

**Comment by ellonde** [comment]
I would presume it works for any task where you have access to your logits
